pkg/k8s/client.go

```diff
@@ -32,7 +32,16 @@ func NewClientset(configPath string, inCluster bool, hostname string) (*kubernet
 
 func restConfig(kubeconfig string, inCluster bool) (*rest.Config, error) {
 	if kubeconfig != "" && !inCluster {
-		return clientcmd.BuildConfigFromFlags("", kubeconfig)
+		cfg, err = clientcmd.BuildConfigFromFlags("", kubeConfig)
```

Comment:
I think `cfg, err = clientcmd.BuildConfigFromFlags("", kubeConfig)` should be `cfg, err := clientcmd.BuildConfigFromFlags("", kubeconfig)`